distribution/docs/src/main/resources/content/_dependencyList/ddf-dependency-list.adoc

```diff
@@ -153,26 +153,26 @@
 * org.apache.cxf:cxf-rt-ws-security:jar:3.1.11
 * org.apache.felix:org.apache.felix.configadmin:jar:1.8.14
 * org.apache.felix:org.apache.felix.coordinator:jar:1.0.2
-* org.apache.felix:org.apache.felix.fileinstall:jar:3.6.0
-* org.apache.felix:org.apache.felix.framework:jar:5.6.6
+* org.apache.felix:org.apache.felix.fileinstall:jar:3.5.8
```

Comment:
I thought that this file was automatically generated. Do the versions need to be updated here? @codice/docs @ricklarsen

Reply:
talked to @ricklarsen and there is a way to get the versions but there is manual process involved in formatting and getting it into the file. He said for update a version to just do it manually.